demo.scala

```diff
@@ -0,0 +1,40 @@
+import java.io._
+import Run._
+
+class Point(xc: Int, yc: Int) {
+   var x: Int = xc
```

Comment:
get these dirty mutable variables out of here

Reply:
Ackshually, sometimes it's better to have mutable variables

https://docs.scala-lang.org/overviews/collections/performance-characteristics.html

Reply:
avoid mutable variables in parallelized environments, especially if not necessary

Reply:
It doesn't compile